### DISCUSSION TITLE
Appache ReverseProxy

### DISCUSSION THREAD

**Discussion by jeje689** [ArchiveBox/ArchiveBox#1151] [Q&A] [answered]
Hi,

I have a server who host many applications. I use Apache ReverseProxy to redirect each subfolder to a app for example : 

https://xxxxxxx.xxxxx.net/app1/ -> http://127.0.0.1:10000/
https://xxxxxxx.xxxxx.net/app2/ -> http://127.0.0.1:10001/
....

My apache configuration : 

```
ProxyPass "/app1/" "http://localhost:10000"
ProxyPassReverse "/app1/" "http://localhost:10000"
ProxyPass "/app2/" "http://localhost:10001"
ProxyPassReverse "/app2/" "http://localhost:10001"
```

I want to do the same redirection for ArchiveBox app. But when I type the url "https://xxxxx.xxxx.net/archivebox/", I come automatically with this url "https://xxxx.xxxx.net/public/. My subfolder into the url was automatically removed. My configuration : 

```
ProxyPass "/archivebox/" "http://localhost:8000"
ProxyPassReverse "/archivebox/" "http://localhost:8000"
```

I add and remove "/" to the subfolder and after port but the result is the same.

Can you help me please?

thank you

**Comment by pirate** [comment]
I'm sorry archivebox does not support running under a sub folder, it needs its own dedicated port. For security reasons it's not safe to host archived content under the same domain with other sites, I recommend returning a redirect response from the path you want to send users to archivebox on another port.

**Reply by jeje689** [reply, marked as answer]
Thank you for your quickly reply. I understand

**Comment by jeje689** [comment]
Thank you for your quickly reply. I understand
